packages/dashboards/examples/src/components/prototypes/prototypes.module.ts

```diff
@@ -1,6 +1,7 @@
 import { CommonModule } from "@angular/common";
 import { NgModule } from "@angular/core";
 import { RouterModule } from "@angular/router";
+import { ConfiguratorHeadingService } from "../../../../src/public-api";
```

Comment:
Please change to `import { ConfiguratorHeadingService } from "@nova-ui/dashboards";`

Reply:
done